drivers/SmartThings/matter-switch/src/init.lua

```diff
@@ -923,15 +924,21 @@ local function occupancy_attr_handler(driver, device, ib, response)
 end
 
 local function cumul_energy_exported_handler(driver, device, ib, response)
-  device:set_field(TOTAL_EXPORTED_ENERGY, ib.data.elements.energy.value)
-  device:emit_event(capabilities.energyMeter.energy({ value = ib.data.elements.energy.value, unit = "Wh" }))
+  if ib.data.elements.energy then
+    local watt_hour_value = ib.data.elements.energy.value * CONVERSION_MILLIWATTS_TO_WATTS
```

Comment:
Maybe a nit, but I think it'd be more clear to just `/ 1000` instead of using the global. The constant itself isn't going to change, it's a bit of a misnomer since energy is in mWh not mW, and also I think it's easier to read if it is more explicitly dividing by 1000 than multiplying by a decimal.

Reply:
I think I disagree here honestly- I personally like this better since it makes it clear why this is needed without adding a comment every time, since the global is kinda a comment in itself. It is a misnomer technically, though a small and clear one imo. 

I do conceed it may be good make it more like:
 ```suggestion
    local watt_hour_value = ib.data.elements.energy.value / CONVERSION_MILLIWATTS_TO_WATTS
```
and turn `CONVERSION_MILLIWATTS_TO_WATTS` into 1000- I just liked the math-y feeling of it being a factor product.